app/src/main/res/menu/menu_user_in_domain.xml

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<menu
```

Comment:
WARNING: [UnusedResources] The resource `R.menu.menu_user_in_domain` appears to be unused
 Unused resources make applications larger and slow down builds.